src/jit/rationalize.cpp

```diff
@@ -93,14 +93,33 @@ void Rationalizer::RewriteSIMDOperand(LIR::Use& use, bool keepBlk)
         return;
     }
 
-    // If we have GT_IND(GT_LCL_VAR_ADDR) and the GT_LCL_VAR_ADDR is TYP_BYREF/TYP_I_IMPL,
-    // and the var is a SIMD type, replace the expression by GT_LCL_VAR.
+    // If we have GT_IND(GT_LCL_VAR_ADDR) and the var is a SIMD type,
+    // replace the expression by GT_LCL_VAR or GT_LCL_FLD.
     GenTree* addr = tree->AsIndir()->Addr();
-    if (addr->OperIsLocalAddr() && comp->isAddrOfSIMDType(addr))
+    if (addr->OperIs(GT_LCL_VAR_ADDR) && comp->lvaGetDesc(addr->AsLclVar())->lvSIMDType)
     {
         BlockRange().Remove(tree);
 
-        addr->SetOper(loadForm(addr->OperGet()));
```

Comment:
`genTreeOps loadForm(genTreeOps addrForm)` is unused after that change, could you please delete it?

Reply:
Cool, didn't notice it's no longer need. Thanks!